qt/widgets/common/src/DataProcessorUI/GenerateNotebook.cpp

```diff
@@ -9,9 +9,9 @@
 #include <boost/regex.hpp>
 #include <boost/tokenizer.hpp>
 #include <fstream>
+#include <iostream>
```

Comment:
Is `iostream` necessary? It's a heavy include and I see that `<sstream>` & `<fstream>` are already included.